pkg/validations/validation_engine.go

```diff
@@ -52,13 +52,17 @@ func fileExists(filename string) bool {
 }
 
 func (ve *validationEngine) LoadConfig(path string) error {
-	v := viper.New()
-
 	if !fileExists(path) {
 		log.Info(fmt.Sprintf("config file %s does not exist. Use default configuration", path))
-		path = ""
+		// legacy disabled checks
+		ve.config.Checks.Exclude = getDisabledChecks()
```

Comment:
I think we can enable everything when the configmap doesn't exist. We will rely on its existence in the OSD clusters.

Reply:
I was only maintaining the compatibility. Previous version was always excluding those checks, but if you think it'd be a good idea to enable everything (but the incompatible ones), I'll change this and the default configmaps yamls on the repo.

Reply:
As discussed yesterday, we don't want to enable everything. Marking as resolved.